src/components/pages/RoleSignup/Applications/Mentor.js

```diff
@@ -415,18 +419,14 @@ const Mentor = ({ dispatch, error, successPage }) => {
                       paddingLeft: '1rem',
                     }}
                   >
-                    <Radio
-                      style={{ margin: '.2rem', width: '100%' }}
-                      value={'yes'}
-                    >
-                      Yes
-                    </Radio>
-                    <Radio
-                      style={{ margin: '.2rem', width: '100%' }}
-                      value={'no'}
-                    >
-                      No
-                    </Radio>
+                    {commitmentArray.map(object => (
```

Comment:
Good usage of a programmatic solution to improve on an earlier verbose, hard-coded version!